nx/source/services/nifm.c

```diff
@@ -21,7 +21,7 @@ Result nifmInitialize(void) {
 
     if (serviceIsActive(&g_nifmSrv))
         return 0;
-		
+        
```

Comment:
Replace with newline, likewise elsewhere.

Reply:
'Replace with newline, likewise elsewhere.'

Reply:
Sorry, didn't spot that one.